controllers/StarHuntController.js

```diff
@@ -242,11 +244,58 @@
         m.set_fillColor('#ffff99');
         var rad = arcsec_to_circle_radius(MARKER_SIZE_ARCSEC, wwt.viewport.Dec);
         m.set_radius(rad);
-        m.setCenter(wwt.viewport.RA * 15, wwt.viewport.Dec); // XXXX
+        m.setCenter(wwt.viewport.RA * 15, wwt.viewport.Dec);
         wwt.wc.addAnnotation(m);
+
+        m.starhunt_is_coord_marker = is_coord;
+        m.starhunt_ra_hours = wwt.viewport.RA; // can't get these back after creation!
+        m.starhunt_dec_deg = wwt.viewport.Dec;
         current_item._markers.push(m);
       }
 
+      $scope.on_report_coordinates = function() {
+        // Figure out the coordinate text to show.
+
+        var text = "";
+
+        if (current_item != null) {
+          for (var i = 0; i < current_item._markers.length; i++) {
+            var m = current_item._markers[i];
+
+            if (m.starhunt_is_coord_marker) {
+              text += util.formatHms(m.starhunt_ra_hours, false, false, false);
+              text += " ";
+              text += util.formatHms(m.starhunt_dec_deg, false, true, false);
+              text += "\n";
```

Comment:
Hi Peter, could we change the colour of the 'coordinate marker' to red by adding m.set_fillColor('ff0000') in this part of the code?

Reply:
Absolutely! It should just be a one- or two-line change as you describe — let me know if anything surprising pops up.